.idea/gradle.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you delete this file, that doesn't seem like it should be part of this PR.

I do believe that @plumpy added all the IDE files explicitly.

Reply:
@fieldju Sorry, I've deleted this by mistake. Recovered it 🙇